### PR TITLE
add Mandarin translation for README.md (close #3170)

### LINKED ISSUE #3170: Translation of README.md into Mandarin
I was hoping that translating the README file to the most spoken language in the world, Mandarin would help with the outreach of the repository. In addition, I would like to work on this as a part of Hacktoberfest.

### PR DESCRIPTION
**Pull request by jagreetdg**
Fixes #3170

### CODE REVIEW

**Comment by jagreetdg** [comment]
I've resolved the invalid links in the README.md and also updated my fork to mirror the Greek Translation Merge in the main Repo in the Link Section. Also it would be helpful if you could add the Hacktoberfest label to this PR as then it would count towards my progress in the same.

**Comment by shahidhk** [comment]
@jagreetdg Since you don't seem to be a native Mandrain speaker, I'd like to ask you if you have used any translation softwares for this PR?

**Comment by jagreetdg** [comment]
No I'm not a native Mandarin speaker but I know Mandarin as my second elective language in my college. Nevertheless I took minor help from translating softwares like Google Translate to assert that all my manual translations were accurate.

**Comment by shahidhk** [comment]
Thank you! Will get this reviewed by a native speaker :smile:

**Comment by jagreetdg** [comment]
Is the review complete or Do I need to make some corrections ?

File: translations/README.chinese.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+* **管理员界面和迁移**: 管理员界面和受Rails启发的架构迁移
+* **Postgres** ❤️: 支持Postgres类型（PostGIS /地理位置等），将视图转换为*图形*，触发具有突变的存储函数或过程
+
+阅读更多 在 [hasura.io](https://hasura.io) 和 [docs](https://docs.hasura.io).
+
+## 目录
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**目录**
+
+- [快速上手:](#快速开始：)
```
Comment (marionschleifer) [inline translations/README.chinese.md]: This link is not working.

File: translations/README.chinese.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+处理中 - [阅读更多](../event-triggers.md).
+
+### 派生数据或数据转换
+
+在Postgres中转换数据或在其上运行业务逻辑以导出可以使用GraphQL Engine查询的另一个数据集 - [阅读更多](https://docs.hasura.io/1.0/graphql/manual/queries/derived-data.html).
+
+## 演示版
+
+在中查看所有示例应用程序
+[community/examples](../community/examples) 目录.
```
Comment (marionschleifer) [inline translations/README.chinese.md]: This link is not working. It was broken on `master` as well. It will probably be fixed when you merge master 🙂

**Comment by marionschleifer** [comment]
@jagreetdg any updates on implementing these changes? 🙂

**Comment by jagreetdg** [comment]
Fixed all the Potential Invalid Links, hope that covers everything.

**Comment by shahidhk** [comment]
This pull request has not been reviewed by a native Mandarin speaker. Hence, there could be some issues.

**Comment by johnbanq** [comment]
I am a native Chinese speaker who is reviewing this translation. I believe a chinese can understand this README, but it won't be pleasant to read. Further polishment and discussion might be needed to make it more readable.

**Comment by shahidhk** [comment]
Closed in favour of https://github.com/hasura/graphql-engine/pull/3639

**Pull request closed without merging**